recipes/deepgraph/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "DeepGraph" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "50b0079141777ec7b8809a3207f78c860cbed86867d5add3f7a31a8840f0bd81" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py33]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
```

Comment:
Change to `numpy x.x` both in the `build` and `run` sections. See:

https://github.com/conda-forge/staged-recipes#4-what-does-numpy-xx-mean